.vscode/launch.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
Are any changes needed to the `list-resources` command? This command is used to generate https://github.com/Azure/azure-resource-manager-schemas/blob/master/generator/resources.json which is used for reporting. @ifeoluwaokunoren FYI

Reply:
Just ran the command `npm run list-resources resources.json`, there's no changes after it.

Reply:
These configurations are just to make it easier for local debugging.

Reply:
Just ran the command `npm run list-resources resources.json`, there's no changes after it. Any more to do? @ifeoluwaokunoren

Reply:
Ideally, we need resources.json to differentiate between the scope of the published schema, since it can either be resourcegroup, subscription, tenant, or managementgroup.  But at this time, we'd break the dependencies on the reporting pipeline, since this is a schema change for resources.json, so for now, let's merge all api-versions without differentiating the scope.

Here's what I think is the ideal structure:
{ 
"Microsoft.AAD/domainServices": [
    { 
      "resourceGroupLevel":
  [
    "2017-01-01",
    "2017-06-01",
    "2020-01-01"
  ]
    },
    {
        "subscriptionLevel":
  [
    "2019-01-01"
  ]
}
  ]
}

Reply:
got it. I will do that.